packages/stryker/src/Sandbox.ts

```diff
@@ -46,11 +46,17 @@ export default class Sandbox {
     }
   }
 
-  public async initialize(): Promise<void> {
+  private async initialize(): Promise<void> {
     await this.fillSandbox();
     return this.initializeTestRunner();
   }
 
+  public static create(options: Config, index: number, files: ReadonlyArray<File>, testFramework: TestFramework | null, coverageInstrumenter: CoverageInstrumenter | null)
+    : Promise<Sandbox> {
+    const sandbox = new Sandbox(options, index, files, testFramework, coverageInstrumenter);
+    return sandbox.initialize().then(() => sandbox);
```

Comment:
So, this returns a promise that so when initialize completes, the sandbox object will be returned?

Reply:
Yes. It was always required to call `initialize` after creating a sandbox. This way, there is no way you can forget doing it.